src/js/pensions/components/IntroductionPage.jsx

```diff
@@ -20,18 +20,23 @@ class IntroductionPage extends React.Component {
         <FormTitle title="Apply for Veterans pension benefits"/>
         <p>Equal to VA Form 21P-527EZ (Application for Pension).</p>
         <SaveInProgressIntro
+          prefillEnabled={this.props.route.formConfig.prefillEnabled}
           pageList={this.props.route.pageList}
-          resumeOnly
+          startClass="no-text-transform"
```

Comment:
Does this need to be a prop? Could we just set `no-text-transform` all the time?

Reply:
Great idea, the class necessitating this has been removed, and instances of non-title-cased text, title cased.